preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -208,7 +208,7 @@ public void discardChanges() {
 
   /**
    * Defines whether the table to debug the undo / redo history should be shown in a dialog when
-   * pressing a key combination or not. <\br> Pressing Ctrl + Shift + H (Windows) or CMD + Shift + H
+   * pressing a key combination or not. <br/> Pressing Ctrl + Shift + H (Windows) or CMD + Shift + H
```

Comment:
this actually needs to be `<br>` else building the JavaDoc fails for me.

Reply:
fixed